geode-core/src/test/java/org/apache/geode/management/internal/security/DiskStoreMXBeanSecurityJUnitTest.java

```diff
@@ -57,7 +56,48 @@ public void setUp() throws Exception {
   }
 
   @Test
-  @ConnectionConfiguration(user = "data-admin", password = "1234567")
+  @ConnectionConfiguration(user = "clusterRead", password = "clusterRead")
+  public void testClusterReadAccess() throws Exception {
+    assertThatThrownBy(() -> bean.flush()).hasMessageContaining(TestCommand.diskManage.toString());
```

Comment:
Thank you regex.  Done and done in commit `97de2f3`.
For what it's worth / future reference, `s/\(\) -> (\w+)\.(\w+)\(\)\)/$1::$2\)` would make 3,232 test replacements, but hits no production files, so that's nice.  Maybe something to add to spotless down the line.